assembl/locale/assembl-v2/de.po

```diff
@@ -1979,6 +2011,30 @@ msgstr "Textwert"
 msgid "administration.thematic"
 msgstr "Thema"
 
+#. [administration.theme.firstColor]
+#. defaultMessage is:
+#. Primary color
+#: messages.json
+#, fuzzy
```

Comment:
Resolve these fuzzies by hand, please :)

Reply:
It's not my change. I think this pr is messy, i will recreate it from scratch